### ISSUE TITLE
`_` being replaced with `-` in slug

### ISSUE THREAD

**Issue by jloh** [sveltia/sveltia-cms#272]
Hey! Apologies for all the issues/noise, happy to cool them off a bit for now if you'd prefer.

When creating new posts using Sveltia I'm getting some weird behaviour with my slugs. My template for slugs + path is this:

```
path: "{{year}}/{{slug}}"
slug: "{{fields.date | date('YYYY-MM-DD')}}_{{fields.date | date('HHmm')}}"
```

And the commit message shows it as rendering correctly, like this:

```
Create Notes “2024-12-11_1213”
```

But the resulting filename is this:

```
content/notes/2024/2024-12-11-1213.md
```

Note that the `_` between the date and time has been replaced with a `-` instead.

All good if underscores aren't available in slugs at the moment, just thought I'd report.

**Comment by kyoshino** [comment]
Checking...

**Comment by kyoshino** [comment]
Fixed in [v0.47.8](https://github.com/sveltia/sveltia-cms/releases/tag/v0.47.8) 🙇🏼

**Comment by jloh** [comment]
Thanks so such a fast fix yet again!
